src/TraceX/TraceXView.xaml.cs

```diff
@@ -1157,7 +1157,7 @@ private void OnExit(object sender, RoutedEventArgs e)
         // Handler for Help -> Manual.
         private void OnOpenManual(object sender, RoutedEventArgs e)
         {
-            string url = "https://docs.microsoft.com/en-us/azure/rtos/tracex/";
+            string url = "https://github.com/eclipse-threadx/rtos-docs";
```

Comment:
I think this should point directly to:
https://github.com/eclipse-threadx/rtos-docs/blob/main/rtos-docs/tracex/overview-tracex.md

Reply:
updated